book/website/collaboration/inclusive-events/inclusive-events-location.md

```diff
@@ -0,0 +1,249 @@
+
+Sensory barriers (airflow, acoustics, lighting, visuals, and smell) can significantly impact the ability your will attendees have to engage in your event. 
+It is helpful to recognize that by choosing a venue that can address some of these barriers, you will be able to host an event that is more comfortable and inclusive of all.
+  
+Finally, consider if your venue is hosted in a space that is "dual purpose". 
+For example, a University Hall may also be used for lectures or by students as a thoroughfare to get to class. 
+If there will be people other than attendees using the space, identify how will you differentiate between staff, attendees, and others who are present in the space. 
+You could use name badges or have staff and organizers in a shared uniform which could help to avoid confusion of an attendee asking a bystander for event assistance. 
+Please also be mindful of how your event will use the spaces provided, for example, if your poster presentation space will also be used to serve food in, 
+try to ensure that food smells are dissipated and plates cleared to support attendees engaging in the session.
+
+When deciding on a venue, try to choose a venue that has the ability to deliver for both accessibility and inclusion needs, and will be a welcoming space to all attendees. 
+
+#### Resources:
+Portsmouth City Council in the UK have compiled a booklet called [Design Guide for Wheelchair-accessible Housing](https://www.portsmouth.gov.uk/wp-content/uploads/2020/05/development-and-planning-design-guide-for-wheelchair-accessible-housing.pdf) 
+that can be used as a reference for assessing venues for their ability to be accessed by wheelchair users. 
+
+(cl-inclusive-events-transport)=
+## Considering Transport Links
+
+When planning an inclusive event for people with lived experience, careful consideration of transportation is essential to ensure that all attendees will be able to access the venue comfortably and safely.
+
+As part of resource planning and depending on the resources available, you may want to consider allocating budget related to the Accessibility fund that can be dedicated to addressing transportation needs. 
+For example, supporting bursaries that cover train tickets and travel for carers and support workers who may need to travel alongside an attendee. 
+
+You should also consider what other access requirements your attendees may require. 
+A multi-day event may involve a social dinner at a separate location which would require the organization of accessible transport options, shuttle services, or potentially a subsidy for attendees with mobility challenges who arrange their own transportation. 
+
+The provision of sufficient parking, both accessible and standard, is important as some participants may prefer travelling by car due to mobility impairments. 
+Try to ensure that there is adequate parking so that all attendees who wish to travel by car are able to park at no disadvantage. 
```

Comment:
This feels like it restates line 88?

Reply:
Good catch. 

I've removed the repeat and merged it with the suggestion to consider parking reimbursements